images/gke-gcloud-auth/README.md

```diff
@@ -0,0 +1,11 @@
+# git image
+
+Use this image when you want to use `gke-gcloud-auth-plugin` and `alpine` as base in a job.
```

Comment:
Do we really need a version with and without git? What is the overhead of including git? I'm not too worried about slightly larger images especially since this layer will be cached on the node and rarely changed.

Reply:
Both crier and deck use images with git, so I wanted to mirror that

Reply:
I think having just 1 image is simpler, and because the delta is less than 1% of the total image size, is the right choice.

Reply:
I've updated the PR to only add 1 new image